pkg/services/accesscontrol/eval/eval.go

```diff
@@ -0,0 +1,8 @@
+package eval
+
+type Evaluator interface {
```

Comment:
I'm not sure about this, but I think it could make it simpler to read when used in the AccessControl package. Wdyt?
```suggestion
type PermissionEvaluator interface {
```